### DISCUSSION TITLE
Stand Up Tuesday April 18th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#61] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Setup PRs
 - Creaeted action to create new discussion every day (didnt work)
 - Worked on Quality Plan
 
**What are your priorities for the next 24 hours?**:
 - PP with @BrandonMagana to create the auth services
 
**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana for his work deploying our environment

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Refactored the Login page with proper NextJS conventions
 - Started NextJS classes with [Gentleman Programming](https://www.youtube.com/@GentlemanProgramming)
 - Set up some GitHub rules
 - Started checking ESLinter config as well as the AirBnB rule set.

**What are your priorities for the next 24 hours?**:
 - Make the Login form interactive and handle user data.
 - Design the sign up on Figma

**Blockers**:
 - No blockers

**Shoutouts**:
 - @everyone who wants to check my PR

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on quallity plan
 
**What are your priorities for the next 24 hours?**:
 - PP with @CarlosCRG19 to set lintern automatically  
 
**Blockers**:
 - Went to dentist and took a lot of time  
 
**Shoutouts**:
 - @everyone you are so handsome
